lib/org/chromium/webidl/ast/Attribute.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl.ast',
+  name: 'Attribute',
+  extends: 'org.chromium.webidl.ast.MemberData',
+  implements: [
+    'org.chromium.webidl.ast.AttributeLike',
+    'org.chromium.webidl.ast.Named',
+    'org.chromium.webidl.ast.Typed',
+  ],
+
+  methods: [
+    function outputWebIDL(o) {
+      if (this.isInherited) o.out('inherited ');
+      if (this.isReadOnly) o.out('readonly ');
+      o.out('attribute ').output(this.type).out(' ').output(this.name).out(';');
```

Comment:
Verbosity isn't a problem if it improves readability. I think it would be ideal to have output(string) and output(WebIDLObject). If that's not possible...maybe outputString and outputObject?